packages/utils/src/db/db.ts

```diff
@@ -703,6 +715,146 @@ export class HoprDB {
       .write()
   }
 
+  /**
```

Comment:
These new functions should be part of another interface/class `NetworkRegistry` instead of directly part of the `db` since they implement highly specific checks. Ideally the db interface stays generic.

Moreover, there seems to be an inconsistency in naming across the PR. We should always refer to it as `NetworkRegistry`, not `Register` or just `Registry`.

Reply:
Naming has been addresses, let's sort out the architecture of the DB in one our architecture calls, and work on improving it.

Reply:
sgtm